pkg/translator/jaeger/cmd/customtransport/buffered_read_transport.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2019 The Jaeger Authors.
```

Comment:
> pkg/translator/jaeger/cmd/customtransport/buffered_read_transport.go

This has nothing to do with `pkg/translator`. All agent code should be under `receiver/jaegerreceiver/internal`